mediacenter/models.py

```diff
@@ -0,0 +1,78 @@
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ideasbox.models import TimeStampedModel
+from search.models import SearchableQuerySet, SearchMixin
+from .utils import guess_type
+
+
+class DocumentQuerySet(SearchableQuerySet, models.QuerySet):
+    def image(self):
+        return self.filter(type=Document.IMAGE)
+
+    def video(self):
+        return self.filter(type=Document.VIDEO)
+
+    def pdf(self):
+        return self.filter(type=Document.PDF)
+
+    def text(self):
+        return self.filter(type=Document.TEXT)
+
+    def audio(self):
+        return self.filter(type=Document.AUDIO)
+
+
+class Document(SearchMixin, TimeStampedModel, models.Model):
+
+    IMAGE = 'image'
+    VIDEO = 'video'
+    PDF = 'pdf'
+    TEXT = 'text'
+    AUDIO = 'audio'
+    OTHER = 'other'
+
+    TYPE_CHOICES = (
+        (IMAGE, _('image')),
+        (AUDIO, _('sound')),
+        (VIDEO, _('video')),
+        (PDF, _('pdf')),
+        (TEXT, _('text')),
+        (OTHER, _('other')),
+    )
```

Comment:
I'm trying to avoid dependencies if I'm not really sure we really need it: this django app will be deployed in a embedded scenario, plus very often with close to no possibility to debug it in case of problem, so I'm really trying to keep the stack under control as much as possible. :)